src/redemptions/redemption.service.spec.ts

```diff
@@ -341,10 +343,53 @@ describe('RedemptionServiceSpec', () => {
         imageCheck,
       );
       const labels = redemptionService.getTransactionLabels(fixture);
-      const acceptableFace =
-        redemptionService.hasOnlyDuplicateFaceFailures(labels);
+      const acceptableFace = redemptionService.hasOnlyBenignWarnings(labels);
       expect(acceptableFace).toBe(true);
     });
+
+    it('should allow LIVENESS_UNDETERMINED warning for success if risk score acceptable', async () => {
+      const fixture = WORKFLOW_RETRIEVE_FIXTURE(
+        'PROCESSED',
+        'CHL',
+        DecisionStatus.WARNING,
+        '1231232',
+        IMAGE_CHECK_FIXTURE('OK'),
```

Comment:
Putting in follow up PR to turn this constructor params into an object so that the function calls aren't bloated with undefineds